### DISCUSSION TITLE
Threshold for how large the changes need to be to be presented to the user

### DISCUSSION THREAD

**Discussion by drupaladmin** [Releem/releem-agent#188] [Feature Requests]
I'd just like to suggest that you implement some threshold for how large the changes need to be to be presented to the user.
 
Take the example below, where the suggestions represent a change of just 0.7%. In addition to that, the suggestion is a decrease, that serves no purpose on a dedicated server (ie. one where the entire RAM is to be allocated to the database). Going from the current RAM usage on the server, it has nothing else to use the memory for. It has already build up a substantial disk buffer of 30gb and no real swap usage. And either way the suggested decrease of 200 megabytes probably wont save anything, even if the server was indeed starved of memory.

<img width="890" alt="Screenshot 2023-08-21 at 11 02 30" src="https://github.com/Releem/mysqlconfigurer/assets/4052657/e665805c-58b7-4622-8bc0-a4f933c66ba4">
<img width="349" alt="Screenshot 2023-08-21 at 11 00 07" src="https://github.com/Releem/mysqlconfigurer/assets/4052657/39e8eebf-05b6-4c68-8e98-9def168bd69d">

**Comment by henrikhj** [comment]
Maybe new suggestions generated by the engine could be held in an escrow until they exceed a defined threshold, like for example 5% if an increase of a buffer, and 10% if a decrease of a buffer (if not otherwise memory constrained). Then when the recommendation exceeds the threshold it would be released from the escrow to the end customer.

**Comment by drupaladmin** [comment]
We've added 3% threshold for a few variables, in a week we'll deploy it to all variables. Please look at results in a few next weeks.

**Comment by drupaladmin** [comment]
Added in the latest release https://github.com/Releem/mysqlconfigurer/releases/tag/1.19.9
